Base/usr/share/man/man1/winlist.md

```diff
@@ -0,0 +1,51 @@
+## Name
+
+winlist - List windows
```

Comment:
I personally feel like the manpage additions should probably be squashed into the commit that adds the utility itself, but others may have other opinions on that.